scripts/send.py

```diff
@@ -126,6 +126,12 @@ def parse_args() -> argparse.Namespace:
     parser = argparse.ArgumentParser()
     parser.add_argument('template', type=Path)
     parser.add_argument('teams_csv', type=Path)
+    parser.add_argument(
```

Comment:
Why aren't we using gmail's mail merge?

Reply:
Because that doesn't support CCing the secondary contacts who've asked to be included. We could in theory do two rounds of sending so they get their own copies, but we're not really set up for that.

Reply:
Secondarily, my intent is to get this script to a point where it can be used fully place of mail-merge as it offers much more flexibility. Ideally eventually getting it behind CI or similar so that we can just write the template emails here and then send them here too -- removing a manual step which currently adds a load of faff.